### PR TITLE
The mergening

### PR DESCRIPTION
**Pull request by mneil**
Compile accepts a single input or folder containing .json rules.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,15 @@
 Grammar for AWS Event Rules:
 <https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns-content-based-filtering.html>
 
-Event Rules are JSON documents that are used to filter other JSON documents.  
-On AWS EventBridge, they are used to filter AWS CloudWatch Events.  
+Event Rules are JSON documents that are used to filter other JSON documents.
```
Comment (3p3r) [inline README.md:6]: there were double-space line breaks at the end of these lines that were accidentally removed.

File: README.md [hunk 2]
```diff
@@ -3,14 +3,15 @@
...
+On AWS EventBridge, they are used to filter AWS CloudWatch Events.
 
 Comparing to other policy formats, Event Rules are limited in terms of features,
 but are easy to understand for non-technical folks and are an extension to JSON.
 
 ![Syntax Highlighting](highlight.png)
 
+
```
Comment (3p3r) [inline README.md:14]: extra whitespace. remove plz.

File: README.md [hunk 3]
```diff
@@ -22,7 +23,7 @@ JSON input and generate the REGO policy.
 $ npm install -g tree-sitter-eventrule
 $ rule2rego --help
 Compiles AWS Event Rule pattern JSON to OPA REGO policy.
-Usage: rule2rego <rule>.json
+Usage: rule2rego [<rule>.json] [folder]
```
Comment (3p3r) [inline README.md:26]: The interface is now kinda awkward. what if the tool accepted multiple jsons on its input and treated them as arrays of rules instead? So you can either pass one or multiple.

like `rule2rego one.json two.json three.json` etc.

Then the expansion of json can be done through the shell.

like `bash -c "rule2rego ./**/*.json"`

File: test/globalSetup.ts [hunk 4]
```diff
@@ -11,13 +11,13 @@ export default async function setup() {
 	// @ts-expect-error - this is how we pass data to tests
 	globalThis.FIXTURES = await Promise.all(
 		glob(`${fixRoot}/*.json`).map((f) =>
-			compile(f).then((policyDocument) => {
+			compile(f).then((policyDocuments) => {
```
Comment (3p3r) [inline test/globalSetup.ts:14]: changing this to `compile(f).then(([policyDocument])` would also save you a `[0]` down below on line 20.

**Comment by mneil** [comment]
Open protest that Rome, while fast, leaves a lot to be desired. In particular the formatting of markdown/json files in this repository have to be handled by a different formatter or kept to style with gut feelings, diffs, and eyeballs on PRs. I think the time wasted going back and forth over a few whitespaces is not worth the 50ms saved formatting a few .ts and .js files.

**Pull request merged**